File: AD-HOC/1285/main.py
from sys import stdin


def max_quantity(m, n):
    quantity = n - m + 1
    for num in map(str, range(m, n + 1)):
        quantity -= len({*num}) != len(num)
    return quantity


for line in stdin:
    m, n = map(int, line.split(' '))
    print(max_quantity(m, n))

File: AD-HOC/1285/main.dart
import 'dart:io';

int maxQuantity(int m, int n) {
  int quantity = n - m + 1;
  for (int num = m; num < n + 1; num++) {
    if (num.toString().split('').toSet().length != num.toString().length) quantity--;
  }
  return quantity;
}

void main() {
  String? line;
  while ((line = stdin.readLineSync()) != null) {
    var [m, n] = line!.split(' ').map(int.parse).toList();
    print(maxQuantity(m, n));
  }
}
